analyzer/runtime/extract.go

```diff
@@ -404,7 +405,27 @@ func ExtractRound(blockHeader nodeapi.RuntimeBlockHeader, txrs []nodeapi.Runtime
 							"err", err2,
 						)
 					}
-					// todo: maybe parse known token methods
+
+					if txr.Result.Ok != nil {
+						// Dead-reckon native token balances.
+						// Native token transfers do not generate events. Theoretically, any call can change the balance of any account,
+						// and we do not have a good way of tracking them; we just query them with the evm_token_balances analyzer.
+						// But heuristically, a call is most likely to change the balances of the sender and the receiver, so we create
+						// a (quite possibly incorrect) dead-reckoned change of 0 for those accounts, which will cause the evm_token_balances analyzer
+						// to re-query their real balance.
+						reckonedAmount := amount.ToBigInt() // Calls with an empty body represent a transfer of the native token.
+						if len(body.Data) != 0 || len(blockTransactionData.SignerData) > 1 {
+							// Calls with a non-empty body have no standard impact on native balance. Better to dead-reckon a 0 change (and keep stale balances)
+							// than to reckon a wrong change (and have a "random" incorrect balance until it is re-queried).
+							reckonedAmount = big.NewInt(0)
+						}
+						registerTokenIncrease(blockData.TokenBalanceChanges, evm.NativeRuntimeTokenAddress, to, reckonedAmount)
+						for _, signer := range blockTransactionData.SignerData {
+							registerTokenDecrease(blockData.TokenBalanceChanges, evm.NativeRuntimeTokenAddress, signer.Address, reckonedAmount)
```

Comment:
Neither. In practice, I don't think we have any txs with multiple signers. If we did, maybe this will do the wrong thing in terms of dead reckoning (or maybe not; IDK what the semantics of `value` are in that case), but at least all the signers will be marked dirty.